x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/create/index.tsx

```diff
@@ -439,7 +439,7 @@ const CreateRulePageComponent: React.FC = () => {
         </EuiFlexGroup>
       </SecuritySolutionPageWrapper>
 
-      <SpyRoute pageName={SecurityPageName.rules} />
+      <SpyRoute pageName={SecurityPageName.rulesCreate} />
```

Comment:
Add `rulesCreate` to `SpyRoute` so we can disable the timeline.

Reply:
Tested this locally -- LGTM 👍 
And thank you for these comments, they are really helpful for code review.